File: src-gen/DangleterreV2/DangleterreV2/Persistence/Models/RoomBook.cs
using System;
using System.Collections.Generic;

namespace DangleterreV2.Persistence.Models
{
    public class RoomBook : IEntity
    {
    	public Guid Id {get; set;}
        public string name {get; set;}
        public Room Room {get; set;} 
        public RoomSchedule plan {get; set;} 
        public NCustomer cus {get; set;} 
    }
}

File: src-gen/DangleterreV2/DangleterreV2/ClientApp/src/api/models/bookings/RoomBook.ts
import {Room} from "../resources/Room"
import {RoomSchedule} from "../schedules/RoomSchedule"
import {NCustomer} from "../customers/NCustomer"

export type RoomBook = {
	id: string
	name: string
	Room: Room
	plan: RoomSchedule
	cus: NCustomer
} 
